msteams-platform/zone-pivot-groups.yml

```diff
@@ -1,12 +1,12 @@
-### YamlMime:ZonePivotGroups
```

Comment:
I wonder if it would be best to create and merge a separate PR, just with these zone-pivot-groups.yml updates before you try to enable zone pivots in an article (per [this note](https://review.docs.microsoft.com/en-us/help/contribute/zone-pivots?branch=master#enable-zone-pivots-in-an-article) in the zone pivot instructions). It might be that the triple hash (rather than single) was not triggering the correct YamlMime type for zone pivots.